package.json

```diff
@@ -91,13 +90,10 @@
     "typescript": "^4.5.5",
     "vite": "4.3.9",
     "vite-plugin-environment": "^1.1.3",
+    "vite-plugin-vuetify": "^1.0.2",
     "vitest": "0.33.0",
-    "vue-class-component": "7.2.6",
     "vuelidate-property-decorators": "1.0.28",
-    "vue-property-decorator": "^9.1.2",
-    "vue-test-utils-helpers": "git+https://github.com/bcgov/vue-test-utils-helpers.git",
-    "vuetify-loader": "^1.9.2",
```

Comment:
Using the package on line 93 now: https://github.com/vuetifyjs/vuetify-loader/tree/master/packages/vite-plugin

Reply:
Is it worth back-porting this to the other UIs?

Reply:
What do you mean Sev?

Reply:
Is it worth changing these packages in the other UIs?

Reply:
Oh, yes I think we need to do so when we're doing the upgrade to the other UIs.

Reply:
Sure, but should we do it ahead of time (in Vue2)?

Reply:
Ohhh, I get it now. That's actually an awesome idea! It could help divide the upgrade process and simplify it a bit.

Reply:
Can you create a few small tickets then? Filings UI, Edit UI, Namerequest UI? Auth Web?

Reply:
Yes, for sure!

Reply:
Sev, I just did some research on this. Apparently, the new plugin [vite-plugin-vuetify](https://github.com/vuetifyjs/vuetify-loader/tree/master/packages/vite-plugin#vite-plugin-vuetify) is exclusively for Vuetify 3. The old one that's removed here is the legacy version `vuetify-loader` (for Vue 2).

Hence, I don't think we can work on this before doing the actual upgrade.

Reply:
OK, good research. I agree with you.